lib/submit/submitSchema.rb

```diff
@@ -35,18 +35,22 @@ def transformPeople(uci, authOrEd, people)
   return if people.empty?
   uci.find!("#{authOrEd}s").build { |xml|
     people.each { |person|
-      xml.send(authOrEd) {
-        if np = person[:nameParts]
-          np[:fname] and xml.fname(np[:fname])
-          np[:mname] and xml.mname(np[:mname])
-          np[:lname] and xml.lname(np[:lname])
-          np[:suffix] and xml.suffix(np[:suffix])
-          np[:institution] and xml.institution(np[:institution])
-          np[:organization] and xml.organization(np[:organization])
-        end
-        person[:email] and xml.email(person[:email])
-        person[:orcid] and xml.identifier(:type => 'ORCID') { |xml| xml.text person[:orcid] }
-      }
+      np = person[:nameParts]
+      if np and np[:organization]
+        xml.organization(np[:organization])
```

Comment:
This is a behavior change. Now, if a person info has organization added, it is treated as an organization not as a person. Please add commends to elaborate.

Reply:
I will add comments.  There is a difference between organization and institution.  Coming from Janeway "organization" only gets added if an author is marked as "corporate" else it's institution. Corporate authors have apparently never worked from the API because accepted the input but encoded it incorrectly.